sway-core/src/declaration_engine/declaration_engine.rs

```diff
@@ -158,26 +135,78 @@ impl DeclarationEngine {
         self.slab.get(*index).expect_struct(span)
     }
 
-    fn de_add_monomorphized_struct_copy(
+    pub(crate) fn de_get_monomorphized_decl(
```

Comment:
Remove the `pub(crate)` because all of these methods are instead accessed via the global methods that access `lazy_static`.